cmd/lifecycle/creator.go

```diff
@@ -280,3 +285,26 @@ func (c *createCmd) registryImages() []string {
 	}
 	return registryImages
 }
+
+func (c *createCmd) platformAPIVersionGreaterThan06() bool {
+	return api.MustParse(c.platform.API()).Compare(api.MustParse("0.7")) >= 0
+}
+
+func (c *createCmd) ReadableRegistryImages() []string {
+	var readableImages []string
+	if c.platformAPIVersionGreaterThan06() {
+		if !c.useDaemon {
+			readableImages = append(readableImages, c.previousImageRef, c.runImageRef)
+		}
+	}
```

Comment:
```suggestion
		if !c.useDaemon {
			readableImages = append(readableImages, c.previousImageRef, c.runImageRef)
		}
```

Reply:
Ooops I forgot to do this here in the creater.

Reply:
Fixed in the following [commit](https://github.com/buildpacks/lifecycle/pull/646/commits/8a82acf4f285ed83dbfb346b7f65f106b548d8aa)